src/query/graphite/native/aggregation_functions.go

```diff
@@ -63,31 +63,86 @@ func diffSeries(ctx *common.Context, series multiplePathSpecs) (ts.SeriesList, e
 		}
 	}
 
-	return combineSeries(ctx, transformedSeries, wrapPathExpr("diffSeries", ts.SeriesList(series)), ts.Sum)
+	return combineSeries(ctx, transformedSeries, wrapPathExpr(DiffSeries, ts.SeriesList(series)), ts.Sum)
 }
 
 // multiplySeries multiplies metrics together and returns the product at each datapoint.
 // If the time series have different intervals, the coarsest interval will be used.
 func multiplySeries(ctx *common.Context, series multiplePathSpecs) (ts.SeriesList, error) {
-	return combineSeries(ctx, series, wrapPathExpr("multiplySeries", ts.SeriesList(series)), ts.Mul)
+	return combineSeries(ctx, series, wrapPathExpr(MultiplySeries, ts.SeriesList(series)), ts.Mul)
 }
 
 // averageSeries takes a list of series and returns a new series containing the
 // average of all values at each datapoint.
 func averageSeries(ctx *common.Context, series multiplePathSpecs) (ts.SeriesList, error) {
-	return combineSeries(ctx, series, wrapPathExpr("averageSeries", ts.SeriesList(series)), ts.Avg)
+	return combineSeries(ctx, series, wrapPathExpr(AverageSeries, ts.SeriesList(series)), ts.Avg)
 }
 
 // minSeries takes a list of series and returns a new series containing the
 // minimum value across the series at each datapoint
 func minSeries(ctx *common.Context, series multiplePathSpecs) (ts.SeriesList, error) {
-	return combineSeries(ctx, series, wrapPathExpr("minSeries", ts.SeriesList(series)), ts.Min)
+	return combineSeries(ctx, series, wrapPathExpr(MinSeries, ts.SeriesList(series)), ts.Min)
 }
 
 // maxSeries takes a list of series and returns a new series containing the
 // maximum value across the series at each datapoint
 func maxSeries(ctx *common.Context, series multiplePathSpecs) (ts.SeriesList, error) {
-	return combineSeries(ctx, series, wrapPathExpr("maxSeries", ts.SeriesList(series)), ts.Max)
+	return combineSeries(ctx, series, wrapPathExpr(MaxSeries, ts.SeriesList(series)), ts.Max)
+}
+
+// lastSeries takes a list of series and returns a new series containing the
+// last value at each datapoint
+func lastSeries(ctx *common.Context, series multiplePathSpecs) (ts.SeriesList, error) {
+	return combineSeries(ctx, series, wrapPathExpr(LastSeries, ts.SeriesList(series)), ts.Last)
+}
+
+// standardDeviationHelper returns the standard deviation of a slice of a []float64
+func standardDeviationHelper(values []float64) float64 {
+	count := float64(0)
+	sum := float64(0)
+
+	for _, value := range values {
+		if !math.IsNaN(value) {
+			sum += value
+			count += 1
+		}
+	}
+	avg := sum / count
+
+	m2 := float64(0)
+	for _, value := range values {
+		if !math.IsNaN(value) {
+			diff := value - avg
+			m2 += diff * diff
+		}
+	}
+
+	variance := m2 / count
+
+	return math.Sqrt(variance)
+}
+func stDevSeries(ctx *common.Context, seriesList multiplePathSpecs) (ts.SeriesList, error) {
```

Comment:
As far as I can tell, this is also not a base graphite function, there's already a `stdev` implementation in `builtin_functions.go`

Reply:
Graphite supports a `stdDevSeries` function, so I need to rename this function that I've implemented, and register it as a real function. 

<img width="738" alt="Screen Shot 2020-09-28 at 8 17 37 AM" src="https://user-images.githubusercontent.com/69990143/94451307-13c50080-0163-11eb-9934-e568552470fa.png">

Graphite docs --> https://graphite.readthedocs.io/en/stable/functions.html#graphite.render.functions.stddevSeries

Reply:
By stdDevSeries func should just call `aggregate`, though.

Reply:
One note though: this documentation says `stddevSeries` is just an alias for calling aggregate with `stddev`. Our implementation of `aggregate()` does not support that because our implementation of `stddev` is not what we need.

`stddevSeries` takes multiple series and calculates one standard deviation value at each step. At step `n`, `stddevSeries` will make a list of every series' `n`th value, and calculate the standard deviation of that list. This is different from how `stddev` works.

Reply:
<img width="749" alt="Screen Shot 2020-10-01 at 10 38 11 AM" src="https://user-images.githubusercontent.com/69990143/94843895-36108580-03d2-11eb-9f2a-b2bbdb2ef413.png">

Reply:
I tried implementing it in the "conventional" aggregation function way (like this):

```
func stddevSeries(ctx *common.Context, series multiplePathSpecs) (ts.SeriesList, error) {
	return combineSeries(ctx, series, wrapPathExpr(StddevSeries, ts.SeriesList(series)), ts.StdDev)
}
```

and it didn't work. There is something about the `combineSeries()` function that doesn't work for StdDev, so I just implemented from scratch.